robottelo/utils/ohsnap.py

```diff
@@ -0,0 +1,56 @@
+import requests
+from box import Box
+
+from robottelo import constants
+from robottelo.exceptions import InvalidArgumentError
+from robottelo.exceptions import RepositoryDataNotFound
+from robottelo.logging import logger
+
+
+def ohsnap_repo_url(
+    ohsnap_repo_host, request_type, product=None, release=None, os_release=None, snap=''
+):
+    """Returns a URL pointing to Ohsnap "repo_file" or "repositories" API endpoint"""
+    if request_type not in ['repo_file', 'repositories']:
+        raise InvalidArgumentError('Type must be one of "repo_file" or "repositories"')
+    if not all(arg for arg in [product, release, os_release]):
```

Comment:
```suggestion
    if not all([product, release, os_release]):
```

Reply:
anyways if you are going to check if these are required arguments then why are you not making them in first place rather adding a check